packages/etsh/Makefile.patch

```diff
@@ -0,0 +1,45 @@
+--- a/Makefile	2023-01-09 17:18:03.884999487 -0300
++++ b/Makefile	2023-01-09 23:08:53.080165062 -0300
+@@ -17,14 +17,14 @@
+ # Choose where and how to install the binaries and manual pages.
+ #
+ DESTDIR?=
+-PREFIX?=	/usr/local
++PREFIX?=	/data/data/com.termux/files/usr
```

Comment:
Do not hardcode the value `$PREFIX`. Use `@TERMUX_PREFIX@` in a patch.

I doubt if this change is needed here though. We already define `PREFIX` env var.